### ISSUE TITLE
No of concurrent session 

### ISSUE THREAD

**Issue by Ikrar-k** [powa-team/powa#121]
Hi,

I have few queries regarding PoWA

1. How to find out no of concurrent user connection in PoWA.
2. Is there a way to check like how much time a session was waiting on a particular wait event or Avg wait events?
3. No of rollback session in PostgreSQL.

**Comment by rjuju** [comment]
Hello,

Unfortunately, most of this is more general monitoring and is out of scope for powa, as the tool is really intended for performance analysis.  So 1 and 3 are not possible, but many tool offer such views.

For 2, that's something no possible to do with postgres infrastructure, as postgres only expose the current wait event if any, but no timing is done.  That's why we chose to use pg_wait_sampling for wait event information, as a high frequency sampling should give a reasonable idea of what are the longest wait events, although of course it's just a statistic approach.  You can see the latest discussion about it at https://www.postgresql.org/message-id/flat/2022793411.102298.1532329482584.JavaMail.jboss%40ep1ml402 for more details.

**Comment by Ikrar-k** [comment]
This is informative. Thank You!! You can close this.

**Comment by rjuju** [comment]
Thanks!
